docs/landing-zone-v2/README.md

```diff
@@ -80,17 +80,21 @@ To deploy this Landing Zone you will need to:
 1. Define environment variables
 
     ```shell
-    export CLUSTER=<cluster-name>
+    export CLUSTER=pdt # <cluster-name>
     export REGION=northamerica-northeast1
-    export PROJECT_ID=<project-id>
-    export LZ_FOLDER_NAME=<env>-<landing zone name>
-    export NETWORK=<vpc-name>
-    export SUBNET=<subnet-name>
-    export ORG_ID=<your_org_id>
+    export CC_PROJECT_RAND=$(shuf -i 0-10000 -n 1) # random string to append to existing project name for new project
+    export BOOT_PROJECT_ID=$(gcloud config list --format 'value(core.project)')
+    export PROJECT_ID=$BOOT_PROJECT_ID-$CC_PROJECT_RAND # globally distinct
+    export LZ_FOLDER_NAME=dev-lz1 #<env>-<landing zone name>
+    export NETWORK=vpc-pdt
+    export SUBNET=vpc-pdt-sn
+    export ORG_ID=$(gcloud projects get-ancestors $BOOT_PROJECT_ID --format='get(id)' | tail -1)
```

Comment:
this gcloud command cannot return a proper value because the config controller project is first created a couple of steps after this one

Reply:
This command uses the current project we are in before we create the CC project

```
To set your Cloud Platform project in this session use “gcloud config set project [PROJECT_ID]”
root_@cloudshell:~$ gcloud config set project kcc-kls
Updated property [core/project].
root_@cloudshell:~ (kcc-kls)$ export CC_PROJECT_RAND=$(shuf -i 0-10000 -n 1)
root_@cloudshell:~ (kcc-kls)$ export BOOT_PROJECT_ID=$(gcloud config list --format 'value(core.project)')
root_@cloudshell:~ (kcc-kls)$ ecoh $BOOT_PROJECT_ID
root_@cloudshell:~ (kcc-kls)$ echo $BOOT_PROJECT_ID
kcc-kls
root_@cloudshell:~ (kcc-kls)$ export PROJECT_ID=$BOOT_PROJECT_ID-$CC_PROJECT_RAND
root_@cloudshell:~ (kcc-kls)$ echo $PROJECT_ID
kcc-kls-7758
root_@cloudshell:~ (kcc-kls)$ export REGION=northamerica-northeast1
root_@cloudshell:~ (kcc-kls)$ export ORG_ID=$(gcloud projects get-ancestors $BOOT_PROJECT_ID --format='get(id)' | tail -1)


later we create the target CC project
root_@cloudshell:~ (kcc-kls)$ gcloud projects create $PROJECT_ID --set-as-default --organization=$ORG_ID
Create in progress for [https://cloudresourcemanager.googleapis.com/v1/projects/kcc-kls-7758]
```

Reply:
Updated readme to add a gcloud config set project early on 
We are working with 2 projects in the readme
the first is the boostrap/starter that we require to get the org and billing id but more essentially to be able to run the following for the org - but requires a services enable on the project that is not the PROJECT_ID (we create later) - this one is BOOT_PROJECT_ID

```
gcloud access-context-manager policies list --organization=${ORG_ID}
needs
gcloud services enable accesscontextmanager.googleapis.com --project=${BOOT_PROJECT_ID}
```

Reply:
For me basing the management project ID based on the current project and a random ID doesn’t work well. It almost seems like a two-step process where I need to have a project created with an ID that I will have to use in the next step where a random ID gets append to the current project. Why not just do something like: PROJECT_ID=my-kcc-cluster-$(shuf -i 0-10000 -n 1). I think it’s one less step if we switch to a current project, extract the ORG and BILLING IDs, then create a new project using PROJECT_ID=my-kcc-cluster-$(shuf -i 0-10000 -n 1). What do you think?